benchmark/run.js

```diff
@@ -1,63 +1,43 @@
 'use strict';
 
-const fs = require('fs');
 const path = require('path');
-const child_process = require('child_process');
-
-var outputFormat = process.env.OUTPUT_FORMAT ||
-                   (+process.env.NODE_BENCH_SILENT ? 'silent' : false) ||
-                   'default';
-
-// If this is the main module, then run the benchmarks
-if (module === require.main) {
-  var type = process.argv[2];
-  var testFilter = process.argv[3];
-  if (!type) {
-    console.error('usage:\n ./node benchmark/run.js <type> [testFilter]');
-    process.exit(1);
-  }
-
-  var dir = path.join(__dirname, type);
-  var tests = fs.readdirSync(dir);
-
-  if (testFilter) {
-    var filteredTests = tests.filter(function(item) {
-      if (item.lastIndexOf(testFilter) >= 0) {
-        return item;
-      }
-    });
-
-    if (filteredTests.length === 0) {
-      console.error('%s is not found in \n %j', testFilter, tests);
-      return;
+const fork = require('child_process').fork;
+const CLI = require('./_cli.js');
+
+const cli = CLI(`usage: ./node benchmark/run.js <type> ...
```

Comment:
There should probably be some explanation (in the help text) about what `<type>` should be exactly...